pdr_backend/sim/sim_engine.py

```diff
@@ -147,7 +165,19 @@ def run_one_iter(self, test_i: int, mergedohlcv_df: pl.DataFrame):
         ut = UnixTimeMs(recent_ut - testshift * timeframe.ms)
 
         # predict price direction
-        prob_up: float = self.model.predict_ptrue(X_test)[0]  # in [0.0, 1.0]
+        if self.ppss.sim_ss.use_own_model is not False:
```

Comment:
is the return type of the property a boolean? If so, there is no need for "is not False". True is True.

Reply:
It could be null or undefined

Reply:
The return value is enforced as bool. How could it be null or undefined?

Reply:
You are right, I'll remove the "is not False" part